test/cpp/end2end/xds/xds_override_host_end2end_test.cc

```diff
@@ -440,6 +449,36 @@ TEST_P(OverrideHostTest, ClusterGoneHostStays) {
       GetAffinityCookieHeaderForBackend(DEBUG_LOCATION, 1, kNumEchoRpcs / 3));
 }
 
+TEST_P(OverrideHostTest, DisabledPerPath) {
+  const std::string kCustomCookieName = "GSSA";
```

Comment:
Using a different cookie name doesn't seem to actually add any value for this test.  I think it can just use `kCookieName`.

If you do keep this for some reason, it can be `absl::string_view` instead of `std::string`.

Reply:
I was working on replicating Richard's config as closely as possible. Would prefer to keep this.

Reply:
As far as I can tell, the cookie name has absolutely no bearing on the bug we're fixing here, so I don't see why it's useful to add all of this additional plumbing for it in the test.  I won't insist on removing it, but IMHO this extra complexity does nothing except distract the reader from what's actually important here.

Reply:
Reverted to default cookie. Thank you for the review!